modules/strap/strap.service.js

```diff
@@ -77,8 +81,8 @@ class StrapService {
       throw new RuleError(STRAP_NOT_FOUND, NOT_FOUND);
     }
 
-    if (foundStrap.image) {
-      await uploadService.deleteFiles([foundStrap.image]);
+    if (foundStrap.images) {
+      await uploadService.deleteFiles([foundStrap.images]);
```

Comment:
Is foundStrap.images an object? The deleteFiles method expects an array containing file names as strings.

Reply:
Done. 
Thanks for tip.